taccsite_cms/static/site_cms/css/src/_imports/trumps/s-header.css

```diff
@@ -124,12 +125,6 @@ Styleguide Trumps.Scopes.Header
 
 /* Search */
 
-.s-header .s-search-bar,
-.s-header .s-search-bar::part(input),
-.s-header .s-search-bar::part(button) {
-  font-size: 16px; /* WARNING: Do not use `rem` until `html { 62.5%; }` */
-}
-
```

Comment:
**Explanation.** The header has long set `font-size: 16px` (for the same reason) for all of its content.